test/integration/install_int_test.go

```diff
@@ -64,7 +64,7 @@ func (suite *InstallIntegrationTestSuite) TestInstall_NoMatches_Alternatives() {
 	defer ts.Close()
 
 	ts.PrepareProject("ActiveState-CLI/small-python", "5a1e49e5-8ceb-4a09-b605-ed334474855b")
-	cp := ts.Spawn("install", "database")
+	cp := ts.Spawn("install", "dataa")
```

Comment:
See https://activestatef.atlassian.net/browse/DX-2805?focusedCommentId=53790 for a description of why this changed.